src/objc_stubs.c

```diff
@@ -0,0 +1,119 @@
+
+/* Compiler implementation of the D programming language
+ * Copyright (c) 2015 by Digital Mars
+ * All Rights Reserved
+ * written by Michel Fortin
+ * http://www.digitalmars.com
+ * Distributed under the Boost Software License, Version 1.0.
+ * http://www.boost.org/LICENSE_1_0.txt
+ * https://github.com/D-Programming-Language/dmd/blob/master/src/objc_stubs.c
+ */
+
+#include "arraytypes.h"
+#include "class.c"
```

Comment:
Is this deliberate? It is causing duplicate symbols when building in VS (not generating a library for the front end). Including `aggregate.h` and `scope.h` instead seems to work, too.